app/javascript/packs/articleModerationTools.js

```diff
@@ -12,7 +12,7 @@ getCsrfToken().then(() => {
       const { initializeActionsPanel } = await import(
         '../actionsPanel/initializeActionsPanelToggle'
       );
-      const { initializeFlagUserModal } = await import('./flagUserModal');
+      // const { initializeFlagUserModal } = await import('./flagUserModal');
```

Comment:
We are no longer using the old FlagUserModal in the ModCenter, so I commented out these initializations.